packages/components/divider/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spark-ui/divider",
-  "version": "0.1.0",
+  "version": "1.0.0-0",
```

Comment:
`1.0.0`?

Reply:
That's the trick that @andresz1 shared, by doing `1.0.0-0` it will publish `1.0.0` instead of `1.0.1` (or `1.1.0`)